src/Esfa.Vacancy.Register.Application/Interfaces/IApprenticeshipSearchService.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Esfa.Vacancy.Register.Application.Interfaces
 {
-    public interface IVacancySearchService
+    public interface IApprenticeshipSearchService
```

Comment:
might as well make the name of this plural as well -> `IApprenticeshipsSearchService`